watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py

```diff
@@ -1398,23 +1398,6 @@ def model6():
 @pytest.mark.unit
 def test_elec_properties_errormsg(model6):
     m = model6
-    with pytest.raises(
-        ConfigurationError,
-        match="Charge data was not provided for N. Provide the missing charge data to the charge argument.",
-    ):
-        m[0].fs.properties = MCASParameterBlock(
-            solute_list=["Na_+", "Cl_-", "N"],
-            mw_data={"H2O": 0.018, "Na_+": 0.023, "Cl_-": 0.0355, "N": 0.01},
-            charge={"Na_+": 1, "Cl_-": -1},
-        )
-    with pytest.raises(
-        ConfigurationError,
-        match="The charge argument was not provided while instantiating the MCAS property model. Provide a dictionary with solute names and associated charge as keys and values, respectively",
-    ):
-        m[0].fs.properties = MCASParameterBlock(
-            solute_list=["Na_+", "Cl_-", "N"],
-            mw_data={"H2O": 0.018, "Na_+": 0.023, "Cl_-": 0.0355, "N": 0.01},
```

Comment:
minor observation - mw for N is 0.014

Reply:
True - although I wasn't too careful when doing this since the tests are really intended to test functionality rather than true values.